### ISSUE TITLE
HDFC portal issue with adguard browser extention enabled [ticket# 916920]

### ISSUE THREAD

**Issue by SM-03** [AdguardTeam/AdguardBrowserExtension#2845]
Adguard Chrome browser extension Version 4.3.53
TSWebExtension v1.0.24
TSUrlFilter v2.2.18
Scriptlets v1.10.25
ExtendedCss v2.0.56

OS: Windows 11 Pro 23H2 22631.3737 x64

Problem since: last 1 month

Issue Details: 
Every time I try to log into my HDFC net banking portal (netportal.hdfcbank.com) after logging in the portal says the following 

"You've Been Logged Out

For security reasons, we have logged you out of HDFC Bank NetBanking. We do this when you refresh/move back on the browser on any NetBanking page."

I turned off all the browser extensions & enabled them one by one to find out which one was causing this issue, it turns out that Adguard Chrome browser extension is the one, so reporting it here for a fix.

Currently, I've added netportal.hdfcbank.com to the allowlist so that in this particular domain Adguard remains inactive 

I already raised this issue via email & the ticket ID is 916920, posting the same here. Pls fix this issue.
TIA

**Comment by alexx7311** [comment]
@SM-03 Hello! We are closing this issue as a dup. of https://github.com/AdguardTeam/AdguardFilters/issues/181648 for now.
If our Filters team decides that the issue is related to the extension itself, we'll reopen the issue.

**Comment by SM-03** [comment]
Hi @alexx7311 
as you advised, I performed the tests

1st test
=======
Filters
All Enabled (updated 19th June 2024)

Stealth Mode
All Disabled

Result & Conclusion: Portal Opened. Seems this issue is not related to the filters, rather something related to the stealth mode option(s)


2nd test
=======
Filters
All Enabled (updated 19th June 2024) as the issue is not related to filters (proven by earlier test 1)

Stealth Mode
All Disabled at first
Login to the portal after enabling every option under this stealth Mode one by one since the beginning. 

So, as per the above testing methodology, 
the issue is related to either anyone or with all these below mentioned three Stealth Mode settings

Miscellaneous
-Hide Referrer from third parties
-Hide Referrer from third parties
-Remove X-Client-Data header

Can't figure out is it only one option out of these three or all these three, Maybe you or anyone else can better enlighten me

And, you were waiting for my response last I checked the status yesterday, the bot closed this issue today just 2hrs ago
So now that  I'm sharing my feedback on the test you asked me to perform, I hope you'll open this again.
